src/model/danbooru.rs

```diff
@@ -54,10 +54,14 @@ pub struct DanbooruPost {
 #[derive(Serialize, Deserialize, Debug, Clone)]
 #[serde(rename_all = "lowercase")]
 pub enum DanbooruRating {
-    E,
-    Q,
-    S,
-    G,
+    #[serde(rename = "e")]
+    Explicit,
+    #[serde(rename = "q")]
+    Questionable,
+    #[serde(rename = "s")]
+    Sensitive,
+    #[serde(rename = "g")]
+    General,
```

Comment:
I must have been quite tired of not seeing literally the [**first**](https://serde.rs/field-attrs.html#rename) item in serdes field documentation when I first implemented the Danbooru stuff.

Reply:
No worries, happens a lot  :+1: